docs/contributing/hacking.md

```diff
@@ -85,10 +85,10 @@ $ make clean
 
 
 [docker]: https://www.docker.com/
-[install]: ../install.md
+[install]: ../install-minikube.md
```

Comment:
Would this - https://github.com/Azure/draft/blob/master/docs/quickstart.md#install-and-configure-helm be a better section to point for installing Helm?

Reply:
Thanks for your review.

I think you're right. 

I think the text where the link is referenced should get edited:
At line 13 there is:
`- [helm][], using the same version as recommended in the [installation guide][install].`

There is no recommended version on neither one of the two references if I'm right?

I propose to change this to:
`- [helm][], see the [quickstart guide][quickstart] for installing helm`
Or to add a general reference at this section to the quickstart guide?

What do you think?